ReSharper.FSharp/test/data/features/quickFixes/convertTupleToElements/Large tuple.fs

```diff
@@ -0,0 +1,3 @@
+let f (x: int array) = ()
+
+f [|1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22{caret}|]
```

Comment:
Any reason this particular length was chosen? Does it add something over the smaller tests?

Reply:
just checking that the regex recognizes looped variable names
![image](https://user-images.githubusercontent.com/26364714/103310550-eb31f880-4a28-11eb-9bc9-354b41646f22.png)

Reply:
Cool, thanks.